src/util/BaseLoginRouter.js

```diff
@@ -211,6 +217,7 @@ function (Okta, Backbone, xdomain, RefreshAuthStateController, Settings, Header,
           success: function () {
             var flashError = this.appState.get('flashError'),
                 model = this.controller.model;
+            this.controller.postRenderAnimation();
```

Comment:
we should add this below line 222, i.e. after the oldController has been removed from the dom